src/wallet/rpc/coins.cpp

```diff
@@ -515,6 +515,7 @@ RPCHelpMan listunspent()
                             {"maximumAmount", RPCArg::Type::AMOUNT, RPCArg::DefaultHint{"unlimited"}, "Maximum value of each UTXO in " + CURRENCY_UNIT + ""},
                             {"maximumCount", RPCArg::Type::NUM, RPCArg::DefaultHint{"unlimited"}, "Maximum number of UTXOs"},
                             {"minimumSumAmount", RPCArg::Type::AMOUNT, RPCArg::DefaultHint{"unlimited"}, "Minimum sum value of all UTXOs in " + CURRENCY_UNIT + ""},
+                            {"include_immature_coinbase", RPCArg::Type::BOOL, RPCArg::Default{false}, "Include immature coinbase transactions"}
```

Comment:
yep, same concept. If the tx is immature, all its outputs are immature. But let me push this.

Reply:
It's not that a big of an issue, but "transactions" is how it is used on all other RPCs that have the same flag, therefore this change creates a small inconsistency.

Reply:
yeah @kouloumos, I thought about it as well. Ended up changing it because in this RPC command `listunspent`, we only care about returning outputs. So, kinda make sense to use the UTXO term here.

Either way is fine for me.

Reply:
Agree, I didn't think it through.